src/multiparser.js

```diff
@@ -159,6 +163,62 @@ function fromHtmlParser2(path, options) {
   }
 }
 
+function wrapCssTemplate(quasisDoc, expressionDocs) {
+  const allReplaced = replacePlaceholders(quasisDoc, expressionDocs);
+  if (!allReplaced) {
+    throw new Error("Couldn't insert all the expressions");
```

Comment:
You are right, let's just leave it like this. If it becomes an issue, people will report it and we'll figure out a solution.